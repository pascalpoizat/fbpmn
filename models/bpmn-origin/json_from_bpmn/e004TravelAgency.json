{
    "isInterrupt": {},
    "catN": {
        "Task_1q91vog": "SendTask",
        "IntermediateThrowEvent_0neineb": "CatchMessageIntermediateEvent",
        "ExclusiveGateway_0i09ijx": "AndGateway",
        "IntermediateThrowEvent_12d113r": "CatchMessageIntermediateEvent",
        "ExclusiveGateway_192ovii": "XorGateway",
        "StartEvent_1": "NoneStartEvent",
        "StartEvent_1f3jj6d": "NoneStartEvent",
        "ExclusiveGateway_1dc5v3z": "XorGateway",
        "TravelAgency_": "Process",
        "EndEvent_0u6deep": "CatchMessageIntermediateEvent",
        "Task_07u875a": "ReceiveTask",
        "Task_1bn6n5q": "SendTask",
        "EndEvent_055yt9k": "NoneEndEvent",
        "EndEvent_10gqkzy": "TerminateEndEvent",
        "Customer_": "Process",
        "Task_002ndsu": "SendTask",
        "Task_1ne4gpy": "SendTask",
        "IntermediateThrowEvent_0xjpikb": "CatchMessageIntermediateEvent",
        "Task_1v9s881": "SendTask",
        "ExclusiveGateway_0wgdt1i": "XorGateway"
    },
    "catE": {
        "SequenceFlow_0bgaa0c": "NormalSequenceFlow",
        "SequenceFlow_03104wi": "NormalSequenceFlow",
        "SequenceFlow_0rfye55": "NormalSequenceFlow",
        "SequenceFlow_0sfyd5z": "NormalSequenceFlow",
        "SequenceFlow_13z4ilm": "NormalSequenceFlow",
        "SequenceFlow_0mdvaai": "NormalSequenceFlow",
        "SequenceFlow_016h32p": "NormalSequenceFlow",
        "SequenceFlow_1uwq0b6": "NormalSequenceFlow",
        "MessageFlow_1m551dh": "MessageFlow",
        "MessageFlow_1yfhhru": "MessageFlow",
        "SequenceFlow_0bztvlk": "NormalSequenceFlow",
        "SequenceFlow_0k1iene": "NormalSequenceFlow",
        "MessageFlow_04an7oz": "MessageFlow",
        "SequenceFlow_0b34324": "NormalSequenceFlow",
        "SequenceFlow_1rma3l8": "NormalSequenceFlow",
        "SequenceFlow_0b6ku63": "NormalSequenceFlow",
        "MessageFlow_1goz1mt": "MessageFlow",
        "MessageFlow_0knd10s": "MessageFlow",
        "SequenceFlow_1h392yr": "NormalSequenceFlow",
        "SequenceFlow_0u8b1w3": "NormalSequenceFlow",
        "SequenceFlow_1h086yg": "NormalSequenceFlow",
        "SequenceFlow_1fn4lqy": "NormalSequenceFlow",
        "SequenceFlow_00r0rkx": "NormalSequenceFlow"
    },
    "containN": {
        "TravelAgency_": [
            "StartEvent_1f3jj6d",
            "ExclusiveGateway_1dc5v3z",
            "Task_1bn6n5q",
            "ExclusiveGateway_0i09ijx",
            "IntermediateThrowEvent_0xjpikb",
            "Task_002ndsu",
            "IntermediateThrowEvent_0neineb",
            "Task_1ne4gpy",
            "EndEvent_10gqkzy"
        ],
        "Customer_": [
            "IntermediateThrowEvent_12d113r",
            "Task_1q91vog",
            "EndEvent_0u6deep",
            "Task_1v9s881",
            "ExclusiveGateway_192ovii",
            "ExclusiveGateway_0wgdt1i",
            "StartEvent_1",
            "Task_07u875a",
            "EndEvent_055yt9k"
        ]
    },
    "messageE": {
        "MessageFlow_1m551dh": "Confirmation",
        "MessageFlow_1yfhhru": "Travel",
        "MessageFlow_04an7oz": "Ticket",
        "MessageFlow_1goz1mt": "Payment",
        "MessageFlow_0knd10s": "Offer"
    },
    "timeInformation": {},
    "targetE": {
        "SequenceFlow_0bgaa0c": "EndEvent_0u6deep",
        "SequenceFlow_03104wi": "Task_1q91vog",
        "SequenceFlow_0rfye55": "IntermediateThrowEvent_0xjpikb",
        "SequenceFlow_0sfyd5z": "ExclusiveGateway_192ovii",
        "SequenceFlow_13z4ilm": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_0mdvaai": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_016h32p": "Task_07u875a",
        "SequenceFlow_1uwq0b6": "Task_1v9s881",
        "MessageFlow_1m551dh": "EndEvent_0u6deep",
        "MessageFlow_1yfhhru": "IntermediateThrowEvent_0xjpikb",
        "SequenceFlow_0bztvlk": "EndEvent_10gqkzy",
        "SequenceFlow_0k1iene": "IntermediateThrowEvent_0neineb",
        "MessageFlow_04an7oz": "IntermediateThrowEvent_12d113r",
        "SequenceFlow_0b34324": "Task_1bn6n5q",
        "SequenceFlow_1rma3l8": "ExclusiveGateway_0wgdt1i",
        "SequenceFlow_0b6ku63": "ExclusiveGateway_0wgdt1i",
        "MessageFlow_1goz1mt": "IntermediateThrowEvent_0neineb",
        "MessageFlow_0knd10s": "Task_07u875a",
        "SequenceFlow_1h392yr": "Task_002ndsu",
        "SequenceFlow_0u8b1w3": "Task_1ne4gpy",
        "SequenceFlow_1h086yg": "EndEvent_055yt9k",
        "SequenceFlow_1fn4lqy": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_00r0rkx": "IntermediateThrowEvent_12d113r"
    },
    "name": "e004TravelAgency",
    "attached": {},
    "containE": {
        "TravelAgency_": [
            "SequenceFlow_1fn4lqy",
            "SequenceFlow_0b34324",
            "SequenceFlow_0mdvaai",
            "SequenceFlow_13z4ilm",
            "SequenceFlow_0rfye55",
            "SequenceFlow_1h392yr",
            "SequenceFlow_0k1iene",
            "SequenceFlow_0u8b1w3",
            "SequenceFlow_0bztvlk"
        ],
        "Customer_": [
            "SequenceFlow_1h086yg",
            "SequenceFlow_00r0rkx",
            "SequenceFlow_03104wi",
            "SequenceFlow_0bgaa0c",
            "SequenceFlow_1uwq0b6",
            "SequenceFlow_0b6ku63",
            "SequenceFlow_0sfyd5z",
            "SequenceFlow_016h32p",
            "SequenceFlow_1rma3l8"
        ]
    },
    "edges": [
        "MessageFlow_0knd10s",
        "MessageFlow_1yfhhru",
        "MessageFlow_1m551dh",
        "MessageFlow_1goz1mt",
        "MessageFlow_04an7oz",
        "SequenceFlow_1h086yg",
        "SequenceFlow_00r0rkx",
        "SequenceFlow_03104wi",
        "SequenceFlow_0bgaa0c",
        "SequenceFlow_1uwq0b6",
        "SequenceFlow_0b6ku63",
        "SequenceFlow_0sfyd5z",
        "SequenceFlow_016h32p",
        "SequenceFlow_1rma3l8",
        "SequenceFlow_1fn4lqy",
        "SequenceFlow_0b34324",
        "SequenceFlow_0mdvaai",
        "SequenceFlow_13z4ilm",
        "SequenceFlow_0rfye55",
        "SequenceFlow_1h392yr",
        "SequenceFlow_0k1iene",
        "SequenceFlow_0u8b1w3",
        "SequenceFlow_0bztvlk"
    ],
    "messages": [
        "Offer",
        "Travel",
        "Confirmation",
        "Payment",
        "Ticket"
    ],
    "sourceE": {
        "SequenceFlow_0bgaa0c": "Task_1v9s881",
        "SequenceFlow_03104wi": "EndEvent_0u6deep",
        "SequenceFlow_0rfye55": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_0sfyd5z": "Task_07u875a",
        "SequenceFlow_13z4ilm": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_0mdvaai": "Task_1bn6n5q",
        "SequenceFlow_016h32p": "ExclusiveGateway_0wgdt1i",
        "SequenceFlow_1uwq0b6": "ExclusiveGateway_192ovii",
        "MessageFlow_1m551dh": "Task_002ndsu",
        "MessageFlow_1yfhhru": "Task_1v9s881",
        "SequenceFlow_0bztvlk": "Task_1ne4gpy",
        "SequenceFlow_0k1iene": "Task_002ndsu",
        "MessageFlow_04an7oz": "Task_1ne4gpy",
        "SequenceFlow_0b34324": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_1rma3l8": "StartEvent_1",
        "SequenceFlow_0b6ku63": "ExclusiveGateway_192ovii",
        "MessageFlow_1goz1mt": "Task_1q91vog",
        "MessageFlow_0knd10s": "Task_1bn6n5q",
        "SequenceFlow_1h392yr": "IntermediateThrowEvent_0xjpikb",
        "SequenceFlow_0u8b1w3": "IntermediateThrowEvent_0neineb",
        "SequenceFlow_1h086yg": "IntermediateThrowEvent_12d113r",
        "SequenceFlow_1fn4lqy": "StartEvent_1f3jj6d",
        "SequenceFlow_00r0rkx": "Task_1q91vog"
    },
    "nodes": [
        "Customer_",
        "TravelAgency_",
        "IntermediateThrowEvent_12d113r",
        "Task_1q91vog",
        "EndEvent_0u6deep",
        "Task_1v9s881",
        "ExclusiveGateway_192ovii",
        "ExclusiveGateway_0wgdt1i",
        "StartEvent_1",
        "Task_07u875a",
        "EndEvent_055yt9k",
        "StartEvent_1f3jj6d",
        "ExclusiveGateway_1dc5v3z",
        "Task_1bn6n5q",
        "ExclusiveGateway_0i09ijx",
        "IntermediateThrowEvent_0xjpikb",
        "Task_002ndsu",
        "IntermediateThrowEvent_0neineb",
        "Task_1ne4gpy",
        "EndEvent_10gqkzy"
    ],
    "nameN": {
        "Task_1q91vog": "Pay Travel",
        "IntermediateThrowEvent_0neineb": "Payment Received",
        "IntermediateThrowEvent_12d113r": "Ticket Received",
        "ExclusiveGateway_192ovii": "is the offer interesting?",
        "StartEvent_1": "Offer Management",
        "StartEvent_1f3jj6d": "Offer Needed",
        "EndEvent_0u6deep": "Booking Confirmed",
        "Task_07u875a": "Check Travel Offer",
        "Task_1bn6n5q": "Make Travel Offer",
        "EndEvent_055yt9k": "Transaction Completed",
        "EndEvent_10gqkzy": "Offer Completed",
        "Task_002ndsu": "Confirm Booking",
        "Task_1ne4gpy": "Order Ticket",
        "IntermediateThrowEvent_0xjpikb": "Booking Received",
        "Task_1v9s881": "Book Travel"
    }
}