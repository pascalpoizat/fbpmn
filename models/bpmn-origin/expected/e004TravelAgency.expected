Working in /tmp/e004TravelAgency.STB1O with 1 worker(s)
[92mtransformation done[0m
<<"Processes=", 2, "Nodes=", 20, "Gateway=", 4, "SF=", 18, "MF=", 5>>
---------- Network01Bag ----------
[X] Prop01Type
     states=437 trans=856 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states= trans= depth=
[X] Prop04MsgSound
     states=437 trans=856 depth=43
---------- Network02FifoPair ----------
[X] Prop01Type
     states=478 trans=903 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=339 trans=596 depth=
[ ] Prop04MsgSound
     states=294 trans=515 depth=
---------- Network03Causal ----------
[X] Prop01Type
     states=624 trans=1099 depth=44
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=624 trans=1099 depth=
[ ] Prop04MsgSound
     states=624 trans=1099 depth=
---------- Network04Inbox ----------
[X] Prop01Type
     states=478 trans=903 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states= trans= depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network05Outbox ----------
[X] Prop01Type
     states=478 trans=903 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=330 trans=582 depth=
[ ] Prop04MsgSound
     states=478 trans=903 depth=
---------- Network06Fifo ----------
[X] Prop01Type
     states=504 trans=921 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=504 trans=921 depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network07RSC ----------
[X] Prop01Type
     states=323 trans=581 depth=43
[ ] Prop02Safe
     states= trans= depth=14
[ ] Prop03Sound
     states= trans= depth=
[ ] Prop04MsgSound
     states= trans= depth=
done.
